docker/base/Dockerfile.ubuntu20.04-1-python

```diff
@@ -30,6 +35,11 @@ RUN cd /opt/build/ && \
 # Remove temporary files
 RUN rm -rf /opt/build/*
 
+# Check that no AVX2/AVX512 instructions are emitted
+RUN if [[ $( objdump -d $(which python3) | grep "%ymm\|%zmm" | wc -l ) != 0 ]] ; then exit 1 ; fi
```

Comment:
Could this be a separate bash function instead that is called in the CI right after building and accepts the filename to be checked as an arg? If we have to change it, it's just currently going to be a fair bit of redundancy and high probability that we miss something